lib/rootApi.js

```diff
@@ -314,6 +305,8 @@ api.remove = callbackify(async (actor, ledgerNodeId, options = {}) => {
       translate: 'owner'
     });
 
+  // delete cache entry even if the update operation fails, the cache will be
```

Comment:
I am failing to find where the cache entry is being deleted? Is this comment relevant?

Reply:
good catch, fixed.